docs/settingUp.md

```diff
@@ -111,6 +113,23 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
+### IntelliJ IDEA <a name="intellij"></a>
+1. Download and install [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/).
+   * You will need an Ultimate Edition license in order to work with Google App Engine in IntelliJ IDEA.
+   * Sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a registered student.
+1. Configure IntelliJ
+   * Click `Configure → Project Defaults → Project Structure`.
+   * Under `Project SDK`, click `New → JDK`. Locate the `Java` folder where you have installed the `JDK 1.7`.
+   *  Select `jdk1.7.0_80` and click `OK`.
+   * Click `OK`.
```

Comment:
IntelliJ's default settings actually follow our standard but I'll add a step to check for that just in case.